tests/c_languages/GNUIndentBearTest.py

```diff
@@ -44,8 +100,43 @@
     settings={"use_spaces": "nope", "max_line_length": "80"})
 
 
-GNUIndentBearGNUIndentWidthTest = verify_local_bear(
+GNUIndentBearWidthTest = verify_local_bear(
```

Comment:
okay doing another commit  for that

2016-06-17 13:13 GMT+01:00 Abhay Raizada <notifications@github.com>:

> In tests/c_languages/GNUIndentBearTest.py
> <https://github.com/coala-analyzer/coala-bears/pull/519#discussion_r67500744>
> :
>
> > @@ -44,8 +100,43 @@
> >      settings={"use_spaces": "nope", "max_line_length": "80"})
> >
> >
> > -GNUIndentBearGNUIndentWidthTest = verify_local_bear(
> > +GNUIndentBearWidthTest = verify_local_bear(
>
> shouldnt this go in another commit?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/519/files/564b35c227bee26b8f18cba050c07dda16120bc3#r67500744>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ9xogW-IHlzU4oZswerNKHRR-z3kKeYks5qMo9-gaJpZM4I02l0>
> .
>